scripts/legacy/retrain_with_expanded_dataset.py

```
@@ -259,10 +259,10 @@ def save_expanded_results(training_history, best_f1, label_encoder, test_data):
         'expanded_samples': len(X_test) + len([x for x in train_data[0]]) + len([x for x in val_data[0]]),
```

Ruff (0.12.2)

Unnecessary list comprehension (rewrite using `list()`)

Rewrite using `list()`

(C416)